lib/erl_interface/include/ei.h

```diff
@@ -188,7 +188,7 @@ extern volatile int __erl_errno;
  * library and when using the library we set a value that we use
  */
 
-#define EI_MAXHOSTNAMELEN 64
+#define EI_MAXHOSTNAMELEN 255
```

Comment:
This is not the value that I had in mind to change to 255. This value is part of the public interface of ei and it should not not changed unless really necessary.